UnityProject/Assets/Scripts/Systems/Construction/WrenchSecurable.cs

```diff
@@ -14,15 +14,14 @@ public class WrenchSecurable : NetworkBehaviour, ICheckedInteractable<HandApply>
 	{
 		private RegisterObject registerObject;
 		private ObjectBehaviour objectBehaviour;
-
 		private HandApply currentInteraction;
 		private string objectName;
 
 		/// <summary>
 		/// Invoked after the anchored state is changed.
 		/// </summary>
-		[NonSerialized]
-		public UnityEvent OnAnchoredChange = new UnityEvent();
+		[NonSerialized] public UnityEvent OnAnchoredChange = new UnityEvent();
```

Comment:
why use an UnityEvent if it is not going to be serialized?

Reply:
Thats just a formatting change, i didnt change that. Whoever made it wanted in nonSerialized i guess?